test/arrays/deitz/part7/test_domain_ref_in_begin.future

```diff
@@ -0,0 +1 @@
+semantic: uses array after it is destroyed
```

Comment:
Looking at this test, I think we probably don't actually want that to work, so I'd be fine with removing it instead of making it a future.  But maybe that's a question for the larger team